codewind-che-sidecar/scripts/kube/deployUtils.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/sh
+
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+set -o pipefail
+
+SCRIPTS_DIR=/scripts
+DEPLOY_DIR=/tmp/$CHE_WORKSPACE_ID
+DEPLOY_STATUS_FILE=$DEPLOY_DIR/deploy-status.json
+
+function updateDeployStatus() {
+    key=$1
+    file=$2
```

Comment:
Will `file` ever be something other than `$DEPLOY_STATUS_FILE`? If not we can just remove this variable and use `$DEPLOY_STATUS_FILE`. That'll simplify calls to this function as well since we won't have to keep passing in `$DEPLOY_STATUS_FILE` as an arg.

Reply:
Not right now. I suppose it might be useful in the future to pass in other templated yaml files. I'm flexible either way though

Reply:
Cool, will leave that with you.